small/small.h

```diff
@@ -40,6 +40,8 @@
 extern "C" {
 #endif /* defined(__cplusplus) */
 
+#define DEFAULT_MEMPOOL_ALIGNMENT sizeof(intptr_t)
```

Comment:
1. Please use enum for constants
2. Why do you need this constant?
3. Why do you call it "alignment" while the patch is about alignment>

Reply:
delete this constant, it's used only in test's and in tuple_init function. Used sizeof(intptr_t) directly.